.github/workflows/ci.yml

```diff
@@ -319,26 +385,29 @@ jobs:
         shell: bash
     name: Website
     timeout-minutes: 30
-    runs-on: ubuntu-20.04
-    container: ubuntu:23.04
+    # runs-on: ubuntu-20.04
+    # container: ubuntu:23.04    
+    runs-on: ubuntu-latest
+    container: ubuntu:20.04
     permissions:
       contents: write
     steps:
-      - name: Setup C++
-        uses: alandefreitas/cpp-actions/setup-cpp@v1.8.3
-        id: setup-cpp
-        with:
-          compiler: clang
-          version: 16
-          check-latest: true
+      # - name: Setup C++
+      #   uses: alandefreitas/cpp-actions/setup-cpp@v1.8.3
+      #   id: setup-cpp
+      #   with:
+      #     compiler: clang
+      #     version: 16
+      #     check-latest: true
 
       - name: Install packages
         uses: alandefreitas/cpp-actions/package-install@v1.8.3
         id: package-install
         with:
-          apt-get: libstdc++-12-dev asciidoctor cmake bzip2 git
-          cc: ${{ steps.setup-cpp.outputs.cc }}
-          cxx: ${{ steps.setup-cpp.outputs.cxx }}
+          # apt-get: libstdc++-12-dev asciidoctor cmake bzip2 git
+          apt-get: build-essential asciidoctor cmake bzip2 git
+          # cc: ${{ steps.setup-cpp.outputs.cc }}
```

Comment:
package-install needs to know where the compilers are in case we include vcpkg dependencies

Reply:
But in this case it is not necessary, because we are only including apt-get dependencies. 
Is there any vcpkg dependency that is being installed by package-install internally that is not evident?